packages/client/src/actions/applyAction.ts

```diff
@@ -202,3 +209,28 @@ async function remapBatchActionParams<
 
   return remappedParams;
 }
+
+function remapActionResponse(
+  response: SyncApplyActionResponseV2 | BatchApplyActionResponseV2,
+): ActionEditResponse | undefined {
+  const editResponses = response?.edits;
+  if (editResponses?.type === "edits") {
+    const { edits, ...editResponse } = editResponses;
+    const remappedActionResponse: ActionEditResponse = {
```

Comment:
Not sure if there is a cleaner way to do this

Reply:
Can you elaborate on your concern more?

Reply:
I was a bit worried while that creating the object and casting it to the ActionEditResponse type, which is the client type rather than the wire type, but looking back I think this is the only way and most typesafe way to do this